src/Moderator.cpp

```diff
@@ -69,19 +69,23 @@ unsigned Moderator::get_search_depth(const DomineeringState& state) const {
     unsigned game_moves = state.getNumMoves();
     unsigned depth;
 
+    if (searcher.get_time_left() > 20.0) {
```

Comment:
We kind of just chose that number. If you have a better number go for it!

-----Original Message-----
From: "Naoki Mizuno" <notifications@github.com>
Sent: ‎11/‎8/‎2015 10:02 PM
To: "NigoroJr/uccineering" <uccineering@noreply.github.com>
Cc: "kojsmn" <kojsmn@miamioh.edu>
Subject: Re: [uccineering] Timer rush (#20)

In src/Moderator.cpp:
> @@ -69,19 +69,23 @@ unsigned Moderator::get_search_depth(const DomineeringState& state) const {
>      unsigned game_moves = state.getNumMoves();
>      unsigned depth;
>  
> +    if (searcher.get_time_left() > 20.0) {
Magic number?
—
Reply to this email directly or view it on GitHub.